streams/src/main/java/org/apache/kafka/streams/kstream/internals/StreamsGraphNode.java

```diff
@@ -93,12 +86,12 @@ Integer id() {
         return this.id;
     }
 
-    public void setStreamsTopologyGraph(final StreamsTopologyGraph streamsTopologyGraph) {
-        this.streamsTopologyGraph = streamsTopologyGraph;
+    void setInternalStreamsBuilder(final InternalStreamsBuilder internalStreamsBuilder) {
```

Comment:
In next PR and foreseeable future, I think the `InternalStreamsBuilder` will remain as a constructor parameter, as it will be used in adding graph nodes to any meta-data collections as graph nodes are constructed.  But to be fair, we are still early in the optimizing phase, so everything is subject to change.